.github/workflows/publish.yml

```diff
@@ -0,0 +1,66 @@
+name: Publish Python distributions to PyPI
+
+on:
+  push:
+    branches: ["**"]
+    tags-ignore: ["**"]
+  pull_request:
+  release:
+    types:
+      - published
+jobs:
+  build_wheels:
+    name: Build wheels [${{ matrix.os }}]
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version:
+          - 3.11
+        os:
+          - ubuntu-24.04
+
+    steps:
+      # We do not release exactly the 1.2.1 tag as we need 
+      # to actually release on urdfdom-py
+      - uses: actions/checkout@v4
+        with:
+          repository: 'ros/urdf_parser_py'
+          ref: '492223ce830cafc4dc2dcd98b82bb0eb91511118'
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install pypa/build
+        run: pip install build
+
+      - name: Build wheel and a source tarball
+        run: python -m build --sdist --wheel --outdir dist/ .
+
+      - name: Archive artifacts
+        uses: actions/upload-artifact@v4
+        with:
+          name: dist
+          path: dist
+
+  deploy:
+    name: Deploy to PyPI
+    runs-on: ubuntu-24.04
```

Comment:
```suggestion
    name: Deploy to PyPI
    runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
         python-version:
           - 3.11
         os:
           - ubuntu-24.04
           - macos-latest

```

Reply:
The wheel is pure python, so I do not think we do not need to publish a different one for each os, just uploading it on Linux is fine and will produce a package that should work fine also on macos.

Reply:
ok! I wasn't sure about this, but you're probably right